test/expected/sql_query_results_optimized.out

```diff
@@ -369,19 +369,20 @@ FROM hyper_1_tz GROUP BY t ORDER BY t DESC limit 2;
 
 EXPLAIN (costs off) SELECT time_bucket(10, time) t, avg(series_0), min(series_1), avg(series_2)
 FROM hyper_1_int GROUP BY t ORDER BY t DESC limit 2;
-                                          QUERY PLAN                                          
-----------------------------------------------------------------------------------------------
+                                                                           QUERY PLAN                                                                            
```

Comment:
Our index scan optimizations break with the case statement. We should figure out how to fix before we implement this change.